node/taskcommand.ts

```diff
@@ -31,7 +31,7 @@ export class TaskCommand {
             for (var key in this.properties) {
                 if (this.properties.hasOwnProperty(key)) {
                     var val = this.properties[key];
-                    if (val) {
+                    if (val != null) {
```

Comment:
I'm still not getting why this is needed

```
$ node
> var properties={}
> var val = properties['foo'];
> if (val) { console.log('yep'); } else { console.log('nope'); }
nope
```

So, it seems to do it's job of only running if the key existed.